src/cloudai/cli/handlers.py

```diff
@@ -243,6 +251,11 @@ def handle_verify_all_configs(args: argparse.Namespace) -> int:
     if err:
         return err
 
+    err, hook_tomls = expand_file_list(HOOK_ROOT, glob="**/*.toml")
```

Comment:
I don't think we should fail if HOOK_ROOT doesn't exist. Most of our users won't have it.

Reply:
Updated.